packages/grafana-data/src/utils/anyToNumber.ts

```diff
@@ -0,0 +1,18 @@
+import { toNumber } from 'lodash';
+
+/** Will return any value as a number or NaN */
+export function anyToNumber(value: any): number {
+  if (typeof value === 'number') {
+    return value;
+  }
+
+  if (value === '' || value === null || value === undefined || Array.isArray(value)) {
+    return NaN; // lodash calls them 0
+  }
+
+  if (typeof value === 'boolean') {
+    return value ? 1 : 0;
+  }
+
+  return toNumber(value);
```

Comment:
nit: Could we use Number instead?
```suggestion
  return Number(value);
```

Reply:
not sure, this was only copied code. Think there are some differences with toNumber vs Number constructor